src/applications/public-outreach-materials/libraries/library-filters.js

```diff
@@ -55,9 +70,47 @@ export function libraryCurrent() {
   });
 }
 
+export function libraryPagerGen() {
+  numCards = libraryNumCards();
+
+  if (document.getElementById('pager-nums-insert')) {
+    const diff = pages - activePage;
+    let pagerHtml;
+
+    pagerHtml = `<button id="va-pagination-active-num" class="va-page-numbers">
+    ${activePage}</button>`;
+    if (diff > 1 && (numCards === undefined || numCards > 9)) {
+      pagerHtml += `<a class="pager-numbers" aria-label="Load page
+      ${activePage + 1}" tabindex="${activePage + 1}">${activePage + 1}</a>`;
+    }
+    if (diff > 2 && (numCards === undefined || numCards > 9)) {
+      pagerHtml += `<a class="pager-numbers"" aria-label="Load page
+      ${activePage + 2}" tabindex="${activePage + 2}">${activePage + 2}</a>`;
+    }
+    if (diff > 3 && (numCards === undefined || numCards > 9)) {
+      pagerHtml += `.... <a class="pager-numbers"" aria-label="Load page
+      ${pages}" tabindex="${pages}"> ${pages}</a>`;
+    }
+    document.getElementById('pager-nums-insert').innerHTML = pagerHtml;
+  }
+}
+
+export function libraryReset() {
+  libraryPagerGen();
+  libraryCurrent();
+  libraryCount();
+}
+
 export function libraryFilters(el) {
-  const pages = Math.ceil(cards.length / 10);
-  if (el.srcElement.id === 'pager-next-click' && activePage !== pages) {
+  if (el.srcElement.className === 'pager-numbers') {
+    activePage = el.srcElement.tabIndex;
+    sessionStorage.setItem('pageNum', el.srcElement.tabIndex);
```

Comment:
To keep track of the page, @cvalarida

Reply:
What happens if the user tries to bookmark the second page? I recommend using URL parameters instead. You could encapsulate this in a couple functions: One to set the page, and one to get it. I think it would be a small update.

Reply:
Would you mind if I made that into a task for next sprint? This will take me a while to figure out, @cvalarida

Reply:
Yeah, I mean, I won't let it stop me from approving. Personally, I think it would be good functionality to have, but that's up to y'all to determine if it's worth getting into this PR or another.